README.md

```diff
@@ -15,6 +15,6 @@ $ java Calculate <action-name> <arguments...>
 - `multiply A B` - Multiplies the `A` by `B`
 - `divide A B` - Divides `A` by `B`
 - `mod A B` - Performs modulo operations from integer `A` and `B`
-- `sin rad` - Gets the sin of the input `rad`
-- `cos rad` - Gets the cos of the input `rad`
-- `tan rad` - Gets the tan of the input `rad`
+- `sin rad` - Gets the sin of the inputted value `rad`
+- `cos rad` - Gets the cos of the inputted value `rad`
+- `tan rad` - Gets the tan of the inputted value `rad`
```

Comment:
No I meant, make it clear what "rad" means. 

E.g. "Gets the sin of the inputted value in radians"